### PR TITLE
Script to visualize lr scheduler 

### LINKED ISSUE #13: [FEATURE] Create utility to visualize learning rate scheduler
**Describe the solution you'd like**
Adding utility scripts to visualize learning rates graph when scheduler applied

### PR DESCRIPTION
**Pull request by triwahyuu**
## Type of changes

 

Please check the type of change your PR introduces:
- [x] Bugfix
- [x] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

close #13 

## What is the new behavior?

- adding [`visualize_learning_rate.py` script](https://github.com/nodefluxio/vortex/blob/lr-visualize/scripts/visualize_learning_rate.py)
- fixing all scripts
-

## Checklist

- [ ] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/vortex/blob/master/CHANGELOG.md)

### CODE REVIEW

File: scripts/visualize_learning_rate.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+def visualize(learning_rates):
+    pass
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="update old model checkpoint to the new format")
+    parser.add_argument('CFG', type=str, nargs='?', help="configuration file path (.yml)")
+    parser.add_argument('-c', '--config', required=False, type=str,
+        help="configuration file path (choose either one of this or positional argument)")
+    parser.add_argument('--epochs', required=False, type=int,
```
Comment (alphinside) [inline scripts/visualize_learning_rate.py:66]: I think this args should be parsed from experiment config too

Comment (triwahyuu) [reply scripts/visualize_learning_rate.py:66]: it would be easier if we could override from the config, if we want to experiment with it.  
also it is not required really, and will use the data from the config if not provided

Comment (alphinside) [reply scripts/visualize_learning_rate.py:66]: hoo i see, if not provided by default it fetch from experiment config right? cool

Comment (triwahyuu) [reply scripts/visualize_learning_rate.py:66]: yepp, that's correct

**Comment by alphinside** [comment]
lgtm, will work on the docs update now

**Comment by alphinside** [comment]
please review @triwahyuu @alifahrri

File: scripts/visualize_learning_rate.py [hunk 2]
```diff
@@ -50,7 +50,10 @@ def calculate_lr(lr_config, epochs=100, optimizer_config=None):
     lr_data = []
     for ep in range(epochs):
         optimizer.step()
-        scheduler.step(ep)
+        try:
```
Comment (triwahyuu) [inline scripts/visualize_learning_rate.py:53]: but the deprecation is just a warning right? or is it an error

Comment (triwahyuu) [reply scripts/visualize_learning_rate.py:53]: probably better if we use the version of pytorch to determine which one to call

Comment (alphinside) [reply scripts/visualize_learning_rate.py:53]: when I try using MultiStepLR and visualize it, it shows different behavior

Comment (triwahyuu) [reply scripts/visualize_learning_rate.py:53]: okay, I think it's fine

Comment (alphinside) [reply scripts/visualize_learning_rate.py:53]: This one when scheduler.step() called, the correct behavior 

step_lr_with_burnin

This one whem scheduler.step(epoch) called, wrong behavior

Figure_2

Comment (triwahyuu) [reply scripts/visualize_learning_rate.py:53]: but to unify our base class with the current pytorch behavior, we could copy the original pytorch implementation from here https://github.com/pytorch/pytorch/blob/master/torch/optim/lr_scheduler.py#L24

Comment (triwahyuu) [reply scripts/visualize_learning_rate.py:53]: probably for another issue for this one
